app/Events/UserNotification.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Events;
+
+use Illuminate\Broadcasting\Channel;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Broadcasting\PrivateChannel;
+use Illuminate\Broadcasting\PresenceChannel;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Broadcasting\InteractsWithSockets;
+use Illuminate\Contracts\Broadcasting\ShouldBroadcast;
+
+class UserNotification implements ShouldBroadcast
+{
+    use Dispatchable, InteractsWithSockets, SerializesModels;
+
+    public $notification;
+    public $userId;
+
+    /**
+     * Create a new event instance.
+     *
+     * @return void
```

Comment:
doc thiếu param này chị

Reply:
@quynt-1571 
à cái này chị dựng để chạy thử thôi, còn code chính thức ở pull khác í em, tí merge là đầy đủ đó 
https://github.com/sunh-1294/Training-php-and-vue/pull/65